reflex/state.py

```diff
@@ -1616,7 +1616,15 @@ def dict(
             self._mark_dirty()
 
         base_vars = {
-            prop_name: self.get_value(getattr(self, prop_name))
+            prop_name: self.__fields__[prop_name].field_info.extra.get("initial_value")
+            if initial
+            and not isinstance(
```

Comment:
Can we add a comment on what this is checking, and potentially move it to a helper function?

Reply:
Yes, I will clean it up later, thanks for the quick review!